packages/frontend/app/models/action.ts

```diff
@@ -0,0 +1,27 @@
+import Model, { attr } from '@ember-data/model';
+
+import VoteChain from 'emberclear/models/vote-chain';
+
+export enum ACTION_RESPONSE {
+  NONE = 'none',
+  APPROVED = 'approved',
+  DENIED = 'denied',
+  DISMISSED = 'dismissed',
+}
+
+export default class Action extends Model {
+  @attr() vote!: VoteChain;
+  @attr() response!: ACTION_RESPONSE;
+  @attr() timestamp!: Date;
+
+  get header() {
```

Comment:
display responsibilities usually go in the component instead of the model (which is data focused)

Reply:
moved to the component side of things.